galleries/olof/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gallerix</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <center>
```

Comment:
The center tag is now considered deprecated. Although it may still work, it is expected to be removed in favor of using CSS. I recommend you use CSS instead.